### PR TITLE
Adds zScan method to Mock

### PR DESCRIPTION
**Pull request by robertmarney**
Adds zScan method to Mock

### CODE REVIEW

File: src/M6Web/Component/RedisMock/RedisMock.php [hunk 1]
```diff
@@ -1211,6 +1211,48 @@ public function zunionstore($destination, array $keys, array $options = array())
...
+        $match   = isset($options[ 'MATCH' ]) ? $options[ 'MATCH' ] : '*';
+        $pattern = sprintf('/^%s$/', str_replace(['*', '/'], ['.*', '\/'], $match));
+
+        $iterator = $cursor;
+
+        if (!isset(self::$dataValues[$this->storage][$key]) || $this->deleteOnTtlExpired($key)) {
+            return $this->returnPipedInfo([0, []]);
+        }
+
+        $set = self::$dataValues[ $this->storage ][ $key ];
```
Comment (thirsch) [inline src/M6Web/Component/RedisMock/RedisMock.php:1235]: Just a small fix for the code style.

```suggestion
        $set = self::$dataValues[$this->storage][$key];
```

**Review by thirsch — COMMENTED** [review]
Thanks for the addition, would also appreciate to get the support soon. Just two small code style comments.

**Review by thirsch — APPROVED** [review]

**Comment by thirsch** [comment]
Thanks, unfortunately, it's not enough to get my approve :D

**Review by Oliboy50 — APPROVED** [review]

**Comment by Oliboy50** [comment]
Thank you (both) for your time.
It has been released in [v5.6.0](https://packagist.org/packages/m6web/redis-mock#v5.6.0).

**Pull request merged**